src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.populateFunc = this.populateFunc.bind(this);
+  }
+
+  populateFunc = () => {
+    const numbers = [];
+    for (let i = 1; i < 10; i += 1) {
+      numbers.push(<button type="button">{i}</button>);
+    }
+    return numbers;
+  }
+
+  render() {
+    return (
+      <div className="mainUI">
```

Comment:
- [ ] **Required:** Great job coming this far in the development of this application but while you have rendered all buttons you have missed the styling required of the component

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/198021496-f1d0927d-7fea-4dc4-a745-a233855b7dee.png) | ![image](https://user-images.githubusercontent.com/27709832/198020854-a0251a01-c279-4eac-89c4-44408f0febd9.png)

Kindly style the component to match this design.